docs/developer/testing/TESTING.md

```diff
@@ -6,56 +6,68 @@
 
 <!-- Can we add this as pre run for make test-e2e? -->
 ```bash
-go install -v github.com/onsi/ginkgo/v2/ginkgo
+go install -v -mod=mod github.com/onsi/ginkgo/v2/ginkgo
 ```
 
-### Setup backup storage configuration
+### AWS setup
 
-To get started, the test suite expects 2 files to use as configuration for
-Velero's backup storage. One file that contains your credentials, and another
-that contains additional configuration options (for now, the name of the
-bucket).
+> **Note:** If you are using IBM Cloud, follow this section.
 
-The default test suite expects these files in `/var/run/oadp-credentials`, but
-can be overridden with the environment variables `OADP_CRED_FILE` and
-`OADP_BUCKET_FILE`.
+To get started, you need to provide the following **required** environment variables.
 
-To get started, create these 2 files:
-`OADP_CRED_FILE`:
+| Variable | Description | Default Value | required |
+|----------|----------|---------------|---------------|
+| `OADP_CRED_FILE` | The path to credentials file for backupLocations | `/var/run/oadp-credentials/new-aws-credentials` | true |
+| `OADP_BUCKET` | The bucket name to store backups | `OADP_BUCKET_FILE` file content | true |
+| `CI_CRED_FILE` | The path to credentials file for snapshotLocations | `/Users/drajds/.aws/.awscred` | true |
+| `VSL_REGION` | The region of snapshotLocations | - | true |
+| `BSL_REGION` | The region of backupLocations | `us-east-1` | false |
+| `OADP_TEST_NAMESPACE` | The namespace were OADP will be installed (it must exist prior to run) | `openshift-adp` | false |
+
+The expected format for `OADP_CRED_FILE` and `CI_CRED_FILE` files is:
 ```
 [<INSERT_PROFILE_NAME>]
 aws_access_key_id=<access_key>
 aws_secret_access_key=<secret_key>
 ```
-If you use one profile name different from `default`, also change `BSL_AWS_PROFILE`.
-
-<!-- change to env var? -->
-`OADP_BUCKET_FILE`:
-```
-<bucket_name>
+> **Note:** If you use one profile name different from `default` for backupLocations, also change `BSL_AWS_PROFILE` environment variable. snapshotLocations must use `default` profile name.
```

Comment:
```suggestion
> **Note:** If you use one profile name different from `default` for backupLocations, also change `BSL_AWS_PROFILE` environment variable. snapshotLocations hardcodes to`default` profile name in e2e code.
```

This was changed recently, but obviously do not apply to older versions.

https://github.com/vmware-tanzu/velero/pull/4864/files#diff-c7c5ce3182fe7a4b576c1784d2178f62bee4a61b009ac94d27e24e21631a2437R29
@sseago can maybe confirm?

Reply:
The recent change was snapshot locations can have their own credentials file. I think profile support for VSLs has been around longer. In any case, I think for e2e tests, there's no profile env var, so you'll need to use default.

Reply:
Agree with the changed text, but I was think the hardcoded string comes from here https://github.com/openshift/oadp-operator/blob/master/tests/e2e/scripts/aws_settings.sh#L36